### ISSUE TITLE
The exception when the jsx and mss files have the same name

### ISSUE THREAD

**Issue by proxiper** [nathantreid/meteor-css-modules#17]
Hello again :)

~~When I compose something from the "both" folders, I get this:~~
This was wrong. The exception occurs when the jsx and mss files have the same name. For example: "app.jsx" and "app.mss" or "app.m.css".

```
=> Errors prevented startup:                  

   While processing files with nathantreid:css-modules (for target web.browser):
   packages/mss/css-modules-build-plugin.js:226:10: Cannot read property 'length' of undefined
   at CssModulesBuildPlugin.compileResultSize (packages/mss/css-modules-build-plugin.js:226:10)
   at LRUCache.length (packages/mss/css-modules-build-plugin.js:41:28)
   at LRUCache.set (.meteor/packages/nathantreid_css-modules/.1.0.3.1j6dxyq++os.osx.x86_64+web.browser+web.cordova/plugin.mss.os.osx.x86_64/npm/node_modules/meteor/mss/node_modules/lru-cache/lib/lru-cache.js:166:18)
   at packages/mss/css-modules-build-plugin.js:175:19
   at .meteor/packages/static-html/.1.0.6.1caorm3++os+web.browser+web.cordova/plugin.compileStaticHtmlBatch.os/npm/node_modules/meteor/promise/node_modules/meteor-promise/fiber_pool.js:33:40

```

meteor 1.3.1
nathantreid:css-modules@1.0.3
kadira:flow-router-ssr@3.12.2
mac os 10.11.2

**Comment by akanix42** [comment]
Can you post a repo for this one? I can't seem to reproduce it.

**Comment by proxiper** [comment]
Sure!) It's hard to reproduce because the problem occurs only when you compose something.
https://github.com/proxiper/meteor-css-modules-same-name-bug

If you rename `imports/ui/app/app.mss` to something else and change the importing in the `app.jsx`, then app will work.

Also in this app you can see this issue too https://github.com/nathantreid/meteor-css-modules/issues/18 -- text has classes for font-size and color, but color styles don't exists and text remains black.

Also please notice that styles generated by a scripts despite the fact that the SSR is disabled.

**Comment by akanix42** [comment]
This should be resolved in 1.1.1 - are you still experiencing issues?
